src/main/java/com/weirddev/testme/intellij/template/context/MockitoMockBuilder.java

```diff
@@ -242,32 +257,32 @@ private String deductMatcherTypeMethod(Param param, Language language) {
     /**
      * true - if should stub tested method
      * @param testMethod method being tested
-     * @param testedClassFields fields of owner type being tested
+     * @param testedClass tested class
      */
     @SuppressWarnings("unused")
-    public boolean shouldStub(Method testMethod, List<Field> testedClassFields) {
```

Comment:
please keep the old version of _shouldStub_ at this stage and annotate it as @Deprecated - so backward compatibility for custom templates would be kept. same goes for _shouldVerify_ method